src/styles/container.scss

```
@@ -0,0 +1,9 @@
+.container {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  margin: 50px auto 0;
+  padding: 0 120px;
+  box-sizing: content-box;
```

```suggestion
  box-sizing: border-box;
```
First of all, `content-box` is preset value of the `box-sizing`. Secondly, it's better to use `border-box`, for better and easier calculations